libc/newhdrgen/yaml_to_classes.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
```

Comment:
This one should be kept. Could you change it to `python3` and add execution bit?